src/main/kotlin/org/jetbrains/research/libsl/nodes/variables.kt

```diff
@@ -1,19 +1,29 @@
 package org.jetbrains.research.libsl.nodes
 
+import org.jetbrains.research.libsl.nodes.references.AnnotationReference
 import org.jetbrains.research.libsl.nodes.references.AutomatonReference
 import org.jetbrains.research.libsl.nodes.references.TypeReference
 import org.jetbrains.research.libsl.type.Type.Companion.UNRESOLVED_TYPE_SYMBOL
 import org.jetbrains.research.libsl.utils.BackticksPolitics
 
 enum class ArithmeticUnaryOp(val string: String) {
-    MINUS("-"), INVERSION("!");
+    PLUS("+"), MINUS("-"), INVERSION("!"), INCREMENT("++"), DECREMENT("--");
 
     companion object {
         fun fromString(str: String) = ArithmeticUnaryOp.values().first { op -> op.string == str }
     }
 }
 
+enum class VariableKeyword(val string: String) {
+    VAR("var"), VAL("val"), RESULT(""), ARGUMENT("");
```

Comment:
I think result variable and arguments aren't keywords. If you wish use it to specify a variable kind, please rename the enum to something like `VariableKind` ot `VariableKind`

Also, what is purpose to distinguish `var`s/`val`s from `result`s and arguments?

Reply:
At first I thought, that any variable should have a field which specifies its kind. The result and argument were added in order to distinguish them from variable declarations. I removed `RESULT` and `ARGUMENT.` The variable kind field was deleted from classes which do not require it